docs/pages/docs/custom-schemas/custom-blocks.mdx

```diff
@@ -77,76 +41,108 @@ const Alert = createReactBlockSpec(
 );
 ```
 
-You can see that `createReactBlockSpec` takes two object arguments:
+#### Block Config (`CustomBlockConfig`)
 
-#### `blockConfig`
+The Block Config describes the shape of the Block, it's type and the properties it supports:
 
-This defines the block's type, properties, and content type. It allows BlockNote to know how to handle manipulating the block internally and provide typing.
+```typescript
+type BlockConfig = {
+  type: string;
+  content: "inline" | "none";
+  readonly propSchema: PropSchema;
+};
+```
 
 **`type`**
 
-Defines the name of the block, in this case, `alert`.
+Defines the identifier of the custom block.
 
 **`content`**
 
-As we saw in [Block Objects](/docs/blocks#block-objects), blocks can contain editable rich text which is represented as [Inline Content](/docs/inline-content). The `content` field allows your custom block to contain an editable rich-text field. Since we want to be able to type in our alert, we set it to `"inline"`.
+`inline` if your custom block should support rich text content, `none` if not.
+
+_In the alert demo, we want the user to be able to type text in our alert, we set it to `"inline"`._
 
 **`propSchema`**
 
-This is an object which defines the props that the block should have. In this case, we want the block to have a `type` prop for the type of alert that we want (warning/error/info/success), so we add a `type` key. We also want basic styling options, so we add text alignment and text color from the [Default Block Properties](/docs/block-types#default-block-properties), using `defaultProps`. The value of each key is an object with a mandatory `default` field and an optional `values` field:
+The `PropSchema` specifices the props that the block supports. The props of a block is data that is associated with your Block, and can be used to customize the block's appearance or behavior.
```

Comment:
Typo, should be specifies

Reply:
Also think we can remove "is data that is associated with your Block, and"

Reply:
replaced with

> The `PropSchema` specifies the props that the block supports. Block props (properties) are data stored with your Block in the document, and can be used to customize a block's appearance or behavior.